lightwood/data/splitter.py

```diff
@@ -16,7 +16,7 @@ def splitter(data: pd.DataFrame, k: int, tss: TimeseriesSettings,
 
     if not tss.is_timeseries:
         # shuffle
-        data = data.sample(frac=1, seed=seed if seed is not None else len(data)).reset_index(drop=True)
+        data = data.sample(frac=1, random_state=seed if seed is not None else len(data)).reset_index(drop=True)
```

Comment:
honestly just make seed = 1 in the kwargs default this is too convoluted

Reply:
I'm not sure why you want it to be the len(data) when you can just as easily do something like

```def splitter(...., seed=1):```

Reply:
As an aside - I've made some other changes to this file @George3d6